modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -125,15 +128,11 @@ This field is enabled after you specify an alias.
 +
 The JKS file information is saved in the global secrets group for your organization.
 
-** Option 3: *Import from Secrets Manager* (for advanced users)
-+
-This option is automatically selected for Runtime Fabrics for which inbound traffic was enabled before the PEM and JKS upload options became available. Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Runtime Fabric Load Balancer] for instructions.
+** Option 3: *Import from Secrets Manager* (for advanced users) - THIS SECTION TO BE MOVED TO ANYPOINT SECURITY DOCS!!!
 +
 This option imports a TLS context from the secrets manager, and supports advanced configuration such as creating a TLS context, mutual authentication, selecting ciphers, and selecting TLS versions. 
 +
-To import a TLS context from the secrets manager, you need additional secrets manager permissions. Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Runtime Fabric Load Balancer] for additional information.
-+
-To generate key material for testing purposes and create a TLS context in the secrets manager, follow the instructions provided in Generate a TLS Certificate for Testing later in this topic.
+To import a TLS context from the secrets manager, you need additional secrets manager permissions. Refer to xref:anypoint-security::configure-tls-context-rtf-lb.adoc[Configure a TLS Context for Runtime Fabric Load Balancer] for additional information. To generate key material for testing purposes and create a TLS context in the secrets manager, follow the instructions provided.
```

Comment:
Techincally, you only need additional permissions to upload a TLS context to Secrets Manager. If there's already one there, you don't need any special permissions to import it.

Reply:
Updated this a bit.